exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/DaemonThreadFactory.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.exporter.trace.ocagent;
+
+import com.google.common.util.concurrent.MoreExecutors;
+import java.util.concurrent.ThreadFactory;
+
+/** A lightweight {@link ThreadFactory} to spawn threads in a GAE-Java7-compatible way. */
+final class DaemonThreadFactory implements ThreadFactory {
```

Comment:
I saw this class at least 3 times. What is the plan for this? Also GAE-Java7 is gone.